finagle-redis/src/main/scala/com/twitter/finagle/redis/HyperLogLogCommands.scala

```diff
@@ -0,0 +1,47 @@
+package com.twitter.finagle.redis
+
+import _root_.java.lang.{Long => JLong,Boolean => JBoolean}
```

Comment:
Does it actually need `_root_` prefix? Seems to me just `java.lang` should work fine.